datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemAttachmentPickerViewHolderFactoryEspressoTest.kt

```diff
@@ -376,6 +376,105 @@ class QuestionnaireItemAttachmentPickerViewHolderFactoryEspressoTest {
       .isEqualTo("Document File")
   }
 
+  @Test
+  fun shouldNotDisplayPreviewFromNullAnswer() {
```

Comment:
optional : can we name something like
```suggestion
  fun doNotShowPreviewIfAnswerDoesNotHaveAttachment() {
```

Reply:
sure